### PR TITLE
ErrorPage for invalid URL, file organisation and global style

### LINKED ISSUE #23: ErrorPage for invalid URL
**Description**
As a user, I want to know when the URL is invalid

**Reasons**
We will display ErrorPage when the URL is invalid.

**Acceptance Criteria**
- [x] Writing invalid URL manually causes ErrorPage to display
- [x] Deleting some letters from URL causes ErrorPage to display

**Additional remarks**
- Router has to be modified 
- ErrorPage has to be implemented

### PR DESCRIPTION
**Pull request by mwisnia97**
**Description**

Changes proposed in this pull request:
- Add ErrorPage implementation
- ErrorPage displays when the URL is invalid
- Reusable components moved to folder src/components
- Unique views moved to folder src/views with own components
- Reusable styles moved to folder src/assets/styles
- Add a prettier config file
- Static images moved to the public folder
- KymaHeader button links now to the HomePage

Resolves #23

### CODE REVIEW

File: frontend/src/views/HomePage/components/FeedArea/FeedArea.styles.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ export const FeedArea = styled.main`
   border-radius: 30px;
   background-color: ${({ theme }) => theme.colors.white};
```
Comment (parostatkiem) [inline frontend/src/views/HomePage/components/FeedArea/FeedArea.styles.js:9]: I think there's no point of storing certain colors in the theme. You can as well use `white`. A color named `white` in the theme will always be white, right?

File: frontend/src/views/HomePage/components/UploadImageArea/UploadImage.styles.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ export const StyledUploadImage = styled.section`
   margin: 30px 50px;
   border-radius: 30px;
   background-color: ${({ theme }) => theme.colors.white};
+  border: 1px solid black;
```
Comment (parostatkiem) [inline frontend/src/views/HomePage/components/UploadImageArea/UploadImage.styles.js:11]: Why do you mix the usage of theme and non-theme colors? I don't think it's a good practice.

**Review by parostatkiem — CHANGES_REQUESTED** [review]
Just a few questions from me 👌

File: frontend/src/views/HomePage/components/UploadImageArea/UploadImage.styles.js (hunk 2 above)
Comment (mwisnia97) [reply frontend/src/views/HomePage/components/UploadImageArea/UploadImage.styles.js:10]: Yes, that was an oversight.

File: frontend/src/views/HomePage/components/FeedArea/FeedArea.styles.js (hunk 1 above)
Comment (mwisnia97) [reply frontend/src/views/HomePage/components/FeedArea/FeedArea.styles.js]: We resigned from using certain colors in the theme.

**Review by parostatkiem — APPROVED** [review]

**Review by Lyczeq — APPROVED** [review]
/lgtm

**Pull request merged**